src/clj/rems/poller/email.clj

```diff
@@ -10,10 +10,9 @@
             [rems.config :refer [env]]
```

Comment:
rems.poller.email could also be folded into the new rems.email ns

if the result is too large, perhaps something like rems.email.template or rems.email.generate for the event-to-email logic?